versions.json

```diff
@@ -14,7 +14,7 @@
         ]
     },
     "latest": "1.31",
```

Comment:
FYI @liam-mackie and I ran into this while you're away (Fixed in https://github.com/OctopusDeploy/kubernetes-agent-tools-base/pull/21)
When `latest` is behind the cluster version, we default to pulling the `latest` tag for the worker tools which has implications for rate limits because k8s just always pulled latest images with our config.

Fixed now, but noteworthy while you're the primary k8s updater around here.